.gitmodules

```diff
@@ -0,0 +1,73 @@
+# This is a git-fleximod adapted .gitmodules file.  Any field with a name starting in fx is a git-fleximod
+# specific field.  See https://github.com/ESMCI/git-fleximod for details.
+
+[submodule "cism"]
+path = source_cism
+url = https://github.com/ESCOMP/cism
+fxtag = cism_main_2.01.013
+fxrequired = AlwaysRequired
+fxDONOTUSEurl = https://github.com/ESCOMP/cism
```

Comment:
Is it really fxDONOTUSEurl? I don't see this in the readme doc at https://github.com/ESMCI/git-fleximod. The "DONOTUSE" looks kinda ... suspicious?

Reply:
that's  correct - it was originally called fxurl but people kept trying to update that instead of url.   I will update the documentation, thanks.